index.js

```diff
@@ -164,7 +164,7 @@ const Minesweeper = {
 		const isWon = this.done && !this.aborted
 
 		// todo: re-render only necessary parts
-		let out = (isGameOver ? '😵 ' : '😬 ') + chalk.gray(` ${this.nrOfMines} mines\n`)
+		let out = (isGameOver ? '😵 ' : isWon ? '😎 ' : '😬 ') + chalk.gray(` ${this.nrOfMines} mines\n`)
```

Comment:
Please make this more readable, e.g. by splitting the emoji and the nr of mines part into two assignments.